versions.yaml

```diff
@@ -77,6 +77,7 @@ module-sets:
       - go.opentelemetry.io/collector/receiver/otlpreceiver
       - go.opentelemetry.io/collector/receiver/receiverprofiles
       - go.opentelemetry.io/collector/receiver/receivertest
+      - go.opentelemetry.io/collector/scraper
```

Comment:
Please update https://github.com/open-telemetry/opentelemetry-collector/blob/main/cmd/builder/test/core.builder.yaml as well

Reply:
No need for the moment, since noboday depends on it.